src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -217,12 +348,14 @@ public void discover(@NotNull final RecipeManager recipeManager)
     @Override
     public List<ItemStack> getListOfAllItems()
     {
+        if (allItems.isEmpty()) Log.getLogger().error("getListOfAllItems when empty");
```

Comment:
These things are some debug code that I've sort of half-deliberately left in.  I never saw any of them in testing (suggesting that the timing is good) but there's a remote possibility that they could be useful to indicate problems if they appear in logs with larger installations or other scenarios that I didn't exercise.

Having said that, they probably wouldn't be super useful other than as a basic indicator, since they lack a stack trace; so I'm happy to just remove them if they don't seem sufficiently worthwhile.